### ISSUE TITLE
Unclear note about PointerEvent initialization of attributes to reflect coalesced events

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/pointerevents#374]
As part of the section relating mainly to coalesced events, the spec contains the following note:

> NOTE
> The PointerEvent's attributes will be initalized in a way that best represents all events in the coalesced event list. For example its `movementX` and `movementY` ([POINTERLOCK]) COULD be the sum of those of all the coalesced events.

I'm actually unclear on what this note is trying to say. Right from the start, "The PointerEvent's attributes" ... which pointer event is it referring to? The coalesced "parent" pointer event that has the list of non-coalesced pointer events? If so, is this trying to define (in a non-normative note, but still) what a user agent could do? The spec already says that it isn't trying to define this. Wonder if the whole note should not simply be removed?

If it's to be kept, it needs some disambiguation perhaps

**Comment by patrickhlauke** [comment]
Re-reading this note, it does feel like it's trying to explain how user agents could work out how to generate the "real" dispatched pointer event based on the attributes of the events in the coalesced events list they keep internally. I think this whole note should be removed, as our spec clearly states that it doesn't want to touch the whole topic of how user agents handle/work out coalesced events etc. it also feels a bit out of place here - while it's on the topic of coalesced events, it doesn't explain anything about coalesced events themselves, but about the "real" pointer events.

long story short: I say this needs to be removed. thoughts?

**Comment by smaug----** [comment]
I think I agree. That movementX/Y example does feel a bit odd. (And does some browser actually do that?)

**Comment by flackr** [comment]
> I think I agree. That movementX/Y example does feel a bit odd.

It makes sense to me. It's trying to say that you don't need to pay attention to the coalesced event list because the top-level events will capture what has happened. I.e. movementX/Y should be the movement since the last top-level event that you would have seen sent to your event handler, not the movement since the previous internal coalesced event.

> (And does some browser actually do that?)

Yes. Try https://jsbin.com/dofojij/edit?html,js,output on chrome. You'll see when there are coalesced events, that the movementX/Y of the main event is the sum of the movement of the coalesced events.

**Comment by patrickhlauke** [comment]
but are we trying here to define what/how the coalescing should do? if so, i *think* that's out of our scope
